### DISCUSSION TITLE
does github swag Shipping cost extra?

### DISCUSSION THREAD

**Discussion by andhra21231** [community/community#17791] [GitHub Education]
I Want To Ask About Is Github Swag Shipping Get Cost Extra at Shipment Fee or anything ? Thanks Btw :D

**Comment by juanpflores** [comment]
Hi @andhra21231 👋🏼 
Great question! There is a shipping cost when you buy swag from the GitHub Shop depending on the country the swag will be shipped to. If the question is about the Graduation swag, you shouldn't need to pay for the shipping. Would recommend trying it out!

**Reply by andhra21231** [reply]
Thanks for your answer @juanpflores

**Comment by azizulabedinazmi** [comment]
When I will receive my GitHub graduation sawag? Any Ideas ?

**Reply by Faris0520** [reply]
2-4 month, maybe

**Reply by andhra21231** [reply]
hopefully is delivery in months 🚀
